evaluator.go

```diff
@@ -372,21 +372,21 @@ func (ev *Evaluator) Run(prune []string) error {
 	paramErrs := MultiError{Errors: []error{}}
 
 	errors.Append(ev.RunPhase(MergePhase))
+	paramErrs.Append(ev.RunPhase(ParamPhase))
+	if len(paramErrs.Errors) > 0 {
```

Comment:
Does this mean if we have an error in the merge phase that results in not overriding a param, we get the param error, instead of the merge error?